From d7c0bd4acd935351d97b4515701dbaf00d7526cc Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Tue, 26 Jan 2021 10:19:49 +0800
Subject: [PATCH] phy: phy-rockchip-snps-pcie3: Remove unused reset signal

We have move p30x1 and p30x2 reset signal to corresponding
controller drivers. So we need to remove them in PHY driver
in order not to add misleading debug message.

Change-Id: I5f180bf51a449ce2db6c49f937d3898a61d7c4d9
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 .../phy/rockchip/phy-rockchip-snps-pcie3.c    | 22 -------------------
 1 file changed, 22 deletions(-)

diff --git a/drivers/phy/rockchip/phy-rockchip-snps-pcie3.c b/drivers/phy/rockchip/phy-rockchip-snps-pcie3.c
index c833209cf36f..54cafe48071f 100644
--- a/drivers/phy/rockchip/phy-rockchip-snps-pcie3.c
+++ b/drivers/phy/rockchip/phy-rockchip-snps-pcie3.c
@@ -27,8 +27,6 @@ struct rockchip_p3phy_priv {
 	int mode;
 	struct regmap *phy_grf;
 	struct reset_control *p30phy;
-	struct reset_control *p30x1;
-	struct reset_control *p30x2;
 	struct clk *ref_clk_m;
 	struct clk *ref_clk_n;
 	struct clk *pclk;
@@ -91,12 +89,6 @@ static int rochchip_p3phy_init(struct phy *phy)
 	}
 
 	reset_control_deassert(priv->p30phy);
-	udelay(1);
-	reset_control_assert(priv->p30x1);
-	reset_control_assert(priv->p30x2);
-	udelay(1);
-	reset_control_deassert(priv->p30x1);
-	reset_control_deassert(priv->p30x2);
 
 	return 0;
 err_pclk:
@@ -112,8 +104,6 @@ static int rochchip_p3phy_exit(struct phy *phy)
 	clk_disable_unprepare(priv->ref_clk_m);
 	clk_disable_unprepare(priv->ref_clk_n);
 	clk_disable_unprepare(priv->pclk);
-	reset_control_assert(priv->p30x1);
-	reset_control_assert(priv->p30x2);
 	reset_control_assert(priv->p30phy);
 	return 0;
 }
@@ -163,18 +153,6 @@ static int rockchip_p3phy_probe(struct platform_device *pdev)
 		priv->p30phy = NULL;
 	}
 
-	priv->p30x1 = devm_reset_control_get(dev, "p30x1");
-	if (IS_ERR(priv->p30x1)) {
-		dev_warn(dev, "no p30x1 reset control specified\n");
-		priv->p30x1 = NULL;
-	}
-
-	priv->p30x2 = devm_reset_control_get(dev, "p30x2");
-	if (IS_ERR(priv->p30x2)) {
-		dev_warn(dev, "no p30x2 reset control specified\n");
-		priv->p30x2 = NULL;
-	}
-
 	priv->ref_clk_m = devm_clk_get(dev, "refclk_m");
 	if (IS_ERR(priv->ref_clk_m)) {
 		dev_err(dev, "failed to find ref clock M\n");
-- 
2.35.3

